nixos/modules/services/networking/nix-serve.nix

```diff
@@ -26,6 +26,12 @@ in
         '';
       };
 
+      openFirewall = mkOption {
+        type = types.bool;
+        default = true;
```

Comment:
this should be false. [modules should not open ports in the firewall](https://nixos.org/manual/nixpkgs/stable/#reviewing-contributions-new-modules) unless explicitly requested, the only exception being [sshd](https://github.com/NixOS/nixpkgs/pull/75454#issuecomment-564573908).

Reply:
Oh, I didn't know that. Fixed, thanks :)